### ISSUE TITLE
New version

### ISSUE THREAD

**Issue by thomaspicquet** [rocketgate/rocketgate-php-sdk#29]
Hi!
Can you create a new version that includes the latest changes?
The last tag was in 2019 :)

thanks!

**Comment by willy-opale** [comment]
Yes please that's something that's required when we only accept stable packages, and changes are required for 3DSv2 as far as I'm aware.

**Comment by dlondero** [comment]
@Nick-RG @lucas-rgate could you please tag a new version?

**Comment by thomaspicquet** [comment]
@Nick-RG this is still needed, on github you can create a new release and at the same time tag a new version

**Comment by bradjones1** [comment]
Really I think this is a request for semantic versioning; Packagist can see tags but they don't get published as new versions unless they fit the semver requirements. In the meantime I'm doing something kinda hacky, a la `"rocketgate/sdk": "dev-master#P8.3"`

CC: @Nick-RG @lucas-rgate
